Update openconfig-network-instance.yang (#987)
openconfig/public: doc/network_instance_redistribution.md

In this latter case, the use of an import policy within a particular protocol instance is equivalent to the former - with the only difference being the context in which the policy is defined.   With a table per protocol, an explicit configuration of redistributing routes from protocol A to protocol B is specified.  In the latter case, the target protocol (protocol B) is specified, and a policy is used to select the source protocols from which routes may be imported for advertisement. 

> In both cases, it is important that the destination protocol (protocol B) in this case only place the redistributed routes from the source protocol (protocol A in this case) in its "advertisement RIB view", but do not use those routes in the active/installed/forwarding route selection process and,consequently, do not place them in the FIB (forwarding information base).

Reviewer: Suggested rewording to attempt to simplify the text but keep the same meaning.  
```suggestion
In both cases,  the destination protocol (protocol B) only installs the redistributed routes from the source protocol (protocol A) in its "advertisement RIB view".  The redistributed routes are not installed in the FIB (forwarding information base) and therefore are not used for forwarding.
```

In this latter case, the use of an import policy within a particular protocol instance is equivalent to the former - with the only difference being the context in which the policy is defined.   With a table per protocol, an explicit configuration of redistributing routes from protocol A to protocol B is specified.  In the latter case, the target protocol (protocol B) is specified, and a policy is used to select the source protocols from which routes may be imported for advertisement.

> In both cases, it is important that the destination protocol (protocol B) in this case only place the redistributed routes from the source protocol (protocol A in this case) in its RIB for advertisements, but do not use those routes in the route selection process and place them in the FIB (forwarding information base).

Reviewer 2: ```suggestion
In both cases, it is important that the destination protocol (protocol B) in this case only place the redistributed routes from the source protocol (protocol A in this case) in its "advertisement RIB view", but do not use those routes in the active/installed/forwarding route selection process and do not place them in the FIB (forwarding information base).
```
